Model/Computed/ComputedInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Magento\UpwardConnector\Model\Computed;
+
+use Magento\Upward\DefinitionIterator;
+
+interface ComputedInterface
+{
+    /**
+     * Resolve value to use in upward config
+     *
+     * @param \Magento\Upward\DefinitionIterator $iterator
+     *
+     * @return mixed
```

Comment:
mixed is not a very good return type for an **API** calling code need take care for many edge cases